src/main/java/com/health/healther/review/controller/ReviewController.java

```diff
@@ -0,0 +1,30 @@
+package com.health.healther.review.controller;
+
+import com.health.healther.review.domain.dto.ReviewCreateRequestDto;
+import com.health.healther.review.service.ReviewService;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+
+@Slf4j
+@RequiredArgsConstructor
+@RequestMapping("/review")
+@RestController
+public class ReviewController {
+
+    private final ReviewService reviewService;
+
+    @PostMapping
+    public ResponseEntity createReview(
+            @RequestBody @Valid ReviewCreateRequestDto request
+    ) {
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
```

Comment:
서비스를 호출하는 코드가 빠져 보입니다!
```suggestion
    @PostMapping
    public ResponseEntity<Void> createReview(
            @RequestBody @Valid ReviewCreateRequestDto request
    ) {
        reviewService.createReview(request)
        return ResponseEntity.ok().build();
    }
```